README.md

```diff
@@ -294,6 +294,80 @@ Executing run...
 	Your job took 420 seconds to complete successfully.
 ```
 
+#### Abort single or multiple jobs from CloudOS
+
+Aborts jobs in the CloudOS workspace that are either running or initialising.
```

Comment:
```suggestion
Aborts jobs in the CloudOS workspace that are either running or initialising. It can be used with one or more job IDs provided as a comma separated string using the `--job-ids` parameter.
```